src/wallet.js

```diff
@@ -79,12 +79,31 @@ const wallet = {
       // 1. Remove one or more spaces (or line breaks) before and after the 24 words
       // 2. Substitute more then one space (or line break) for a single space
       newWordsString = words.trim(/\s+/).replace(/\s+/g, ' ');
-      if (newWordsString.split(' ').length !== 24) {
+      const wordsArray = newWordsString.split(' ');
+      if (wordsArray.length !== 24) {
         // Must have 24 words
         return {'valid': false, 'message': 'Must have 24 words'};
       } else if (!Mnemonic.isValid(newWordsString)) {
-        // Invalid sequence of words
-        return {'valid': false, 'message': 'Invalid sequence of words'};
+        // Check if there is a word that does not belong to the list of possible words
+        const wordlist = Mnemonic.Words.ENGLISH;
+        const wordsLowerCase = wordsArray.map((value) => value.toLowerCase());
+        const errorList = [];
+
+        for (let i = 0; i < wordsLowerCase.length; i += 1) {
+          const w = wordsLowerCase[i];
```

Comment:
Why not `for (const word of wordsLowerCase)`?

Reply:
No particular reason. Changed to `for ... of ...`, it's better